db/migrate/20170503172351_add_web_domain_to_accounts.rb

```diff
@@ -0,0 +1,16 @@
+class AddWebDomainToAccounts < ActiveRecord::Migration[5.0]
+  def up
+    add_column :accounts, :web_domain, :string, null: true, default: nil
+
+    ActiveRecord::Base.transaction do
```

Comment:
Yeah, sorry about making you do the migration bit. See my comment on the other file.